genestack_client/scripts/genestack_user_setup.py

```diff
@@ -189,7 +216,11 @@ def run(self):
         users = config.users
         user = users.get(self.args.alias)
         if not user:
-            user = select_user(users, None)  # TODO get current user for shell and command line
+            user = select_user(users)
+
+        if not user.email:
+            print('User without email cannot have password')
+            return
```

Comment:
Can such a user have anything token?
If yes, then `"User without email could be authorized only by token"`